docs/DeveloperGuide.adoc

```diff
@@ -864,6 +864,33 @@ Windows, Linux, Unix, OS-X
 [[private-contact-detail]] Private contact detail::
 A contact detail that is not meant to be shared with others
 
+[[personal-trainer]] Personal trainer::
```

Comment:
Might want to be more specific in defining target user profile. Who are they? What they do? What are their problems? What are their current solutions? What your value proposition that might be better than these current solutions?